inference-engine/src/transformations/include/ngraph_ops/subgraph.hpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/function.hpp>
+#include <ngraph/op/op.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "transformations/snippets/generator.hpp"
+
+namespace ngraph {
+namespace op {
+
+
+// \brief An operation that is implemented by a function.
+class TRANSFORMATIONS_API Subgraph : public Op {
```

Comment:
We already have operations in the nGraph which can contain subgraph. Do we really need this specific op?